docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-micro-managed.adoc

```diff
@@ -90,17 +90,15 @@ To change this, provide java debug options to `cmdOptions`.
 |===
 
 These options can all be provided through either an environment variable or system property.
-The easiest way to provide these to the test instance is through the surefire plugin.
+The easiest way to provide these to the test instance is is by using an `arquillian.xml` file.
 
 [source,XML]
 .Example
 ----
-<plugin>
-    <artifactId>maven-surefire-plugin</artifactId>
+<container qualifier="payara-micro-managed">
```

Comment:
I'd add default="true" to all 3 examples so that they can be copied and will just work. Otherwise people need to remember to add it or set the arquillian.launch system property to the qualifier.
```suggestion
<container qualifier="payara-micro-managed" default="true">
```